src/test/java/ch/liquidmind/inflection/proxy/util/ProxyGeneratorTestUtility.java

```diff
@@ -0,0 +1,67 @@
+package ch.liquidmind.inflection.proxy.util;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+
+import javax.tools.DiagnosticCollector;
+import javax.tools.JavaCompiler;
+import javax.tools.JavaFileObject;
+import javax.tools.StandardJavaFileManager;
+import javax.tools.ToolProvider;
+
+import com.google.common.io.Files;
+
+import ch.liquidmind.inflection.loader.TaxonomyLoader;
+import ch.liquidmind.inflection.model.external.Taxonomy;
+import ch.liquidmind.inflection.model.external.View;
+import ch.liquidmind.inflection.proxy.Proxy;
+import ch.liquidmind.inflection.proxy.ProxyGenerator;
+import ch.liquidmind.inflection.test.TestUtility;
+
+public final class ProxyGeneratorTestUtility
+{
+
+	public static File createProxy(File compiledTaxonomyDir, Taxonomy taxonomy, View view) {
```

Comment:
compiledTaxonomyDir is not being used (wouldn't it be nice if Eclipse warned us about this?)